src/code/photo_album.asm

```diff
@@ -760,19 +760,40 @@ func_028_44DB::
     inc  [hl]                                   ; $44DE: $34
     ret                                         ; $44DF: $C9
 
+offset: macro
+    dw \1
+    db BANK(\1)
+endm
```

Comment:
Maybe this could be an occasion to re-use the `entity_pointer` macro, which has a similar structure (but it's also fine with me like this).